### ISSUE TITLE
Should ignore .gitattributes

### ISSUE THREAD

**Issue by mgedmin** [mgedmin/check-manifest#59]
There's no reason to insist on .gitattributes files being present in source distributions.

**Comment by mgedmin** [comment]
Huh, .gitattributes is [already present](https://github.com/mgedmin/check-manifest/blob/765d81f5002b290039ec3b3c2f287bcc5a4f9830/check_manifest.py#L442) in the default ignore list.  So why does this happen?

```
$ git clone https://github.com/zopefoundation/z3c.autoinclude
$ cd z3c.autoinclude
$ check-manifest
lists of files in version control and sdist do not match!
missing from sdist:
  .gitattributes
```

**Comment by mgedmin** [comment]
Oh.  I had an old version of check-manifest (0.24) installed.

**Comment by mgedmin** [comment]
This was fixed in 0.26, via #50.
